Assets/MixedRealityToolkit/Definitions/Devices/MixedRealityHandTrackingProfile.cs

```diff
@@ -47,7 +47,36 @@ public class MixedRealityHandTrackingProfile : BaseMixedRealityProfile
 
         [SerializeField]
         [Tooltip("If true and the hand mesh is available, try to access the hand mesh from the system. Note: this could reduce performance")]
-        private bool enableHandMeshUpdates = false;
-        public bool EnableHandMeshUpdates => enableHandMeshUpdates;
+        private bool enableHandMeshVisualization = false;
```

Comment:
Might be good to add the [`FormerlySerializedAttribute`](https://docs.unity3d.com/ScriptReference/Serialization.FormerlySerializedAsAttribute.html
) here, so we don't break set-ups that already have this on.